contributed_definitions/NXcontainer.nxdl.xml

```diff
@@ -151,12 +151,14 @@
 			adsorption of the container to be calculated.
 		</doc>
 	</group>
-	<group name="empty_measurement" type="NXentry">
+	<group name="reference_measurement" type="NXentry">
```

Comment:
And this instance has it as a child of a definition element.  I'll adjust the documentation.  Thanks.